scripts/start-op-node.sh

```diff
@@ -20,4 +20,5 @@ exec op-node \
   --metrics.enabled \
   --metrics.addr=0.0.0.0 \
   --metrics.port=7300 \
+  --snapshot=false \
```

Comment:
`t=2023-07-20T08:34:54+0000 lvl=crit msg="Application failed" message="flag provided but not defined: -snapshot"`